staging/src/k8s.io/client-go/discovery/cached_discovery.go

```diff
@@ -30,15 +30,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/version"
-	"k8s.io/client-go/discovery"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
```

Comment:
You may not have this dependency in client-go.  nothing from kube/kube

Reply:
> witched to k8s.io/client-go/kubernetes/scheme

What is it used for?  I expected you to create your own scheme locally like our other clients do.